defaults/main.yml

```diff
@@ -93,6 +93,12 @@ controlhost_maven_mirrors: []
 # When set the given version of Ansible will be installed on the controlhost
 #controlhost_ansible_version: 2.5.6
 
+# Use either 2 for Python2 or 3 for Python3 here
+controlhost_python_version: 3
+
+# Set path to the pip executable, otherwise it will be set depending on controlhost's OS family
+#controlhost_pip_executable: /usr/bin/pip3
```

Comment:
This new role variable needs to be added in the readme as well.

Reply:
I just found this isn't necessary anymore and I will kick it by my next commit. The error I was trying to solve with this variable can easily solved by using `controlhost_packages_pip_upgrade: false`. If pip gets upgraded, it may change the path of its binary, depending on how it is upgraded (`sudo` or not, `python3 -m pip` or `pip3`), at least on CentOS 8. This leads to the error that is documented [over here](https://github.com/pypa/pip/issues/5599).